src/wmnds/components/lists/ordered-list/_ordered-list.scss

```diff
@@ -0,0 +1,21 @@
+.wmnds-ordered-list, ol {
```

Comment:
You might have to remove the `ol` from this line

Reply:
Done